src/assets/scripts/client/airport/AirportWind.js

```diff
@@ -0,0 +1,35 @@
+import AirportModel from './AirportModel';
```

Comment:
In javascript, importing the `AirportModel` here creates a circular reference, which is bad.  Good news is you don't need this import and it can be removed 🎉